### PR TITLE
Fabo/fix safari issue

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
🚀 Deployed on https://5f19c1b3d594358b561e0d0a--lunieio.netlify.app

File: app/src/scripts/google-analytics.js [hunk 1]
```diff
@@ -64,15 +65,25 @@ module.exports.sendEvent = function event(customObject, ...args) {
...
 }
+
+module.exports = {
+  anonymize,
+  deanonymize,
+  enableGoogleAnalytics,
+  disableGoogleAnalytics,
+  sendEvent,
+  track,
+  setGoogleAnalyticsPage
```
Comment (Bitcoinera) [inline app/src/scripts/google-analytics.js:88]: Looks good. How can I test this in Bitrise? I forgot the whole flow there. Maybe I still have bookmarked on my Windows. Not sure

Comment (faboweb) [reply app/src/scripts/google-analytics.js:88]: https://live.browserstack.com/ there is also a chrome extension that will open the current URL right in browserstack

Comment (Bitcoinera) [reply app/src/scripts/google-analytics.js:88]: cool, I will check this later

**Pull request merged**